### PR TITLE
News section now matches the likes section

### PR DESCRIPTION
**Pull request by AbelKel**
**News Section**
* The news section now matches the likes section.

**Bugs**
* None detected so far

**Next PR**
1. I currently have two options for displaying the details of a college news article: 
**Option 1**: Redirect to the news website 
**Option 2**: Create a details view
2. Parse relation optimization for matching users with colleges
3. App translation 

Demos:
https://drive.google.com/file/d/12moDGz6SJc2_FTNeghpglHCxX0-jXKEF/view?usp=sharing

Screenshots: 
**News about Likes**
Screen Shot 2022-07-28 at 2 39 08 PM
**Liked Colleges**
Screen Shot 2022-07-28 at 2 39 31 PM

### CODE REVIEW

File: Capstone/API/APIManager.h
Comment (skhezri) [inline Capstone/API/APIManager.h]: typo
```suggestion
- (void)fetchCollegeNews:(NSString *)collegeWebsite getArrayOfCollgeNews:(void(^)(NSArray *collegeNews, NSError *error))completion;

File: Capstone/API/APIManager.m
Comment (skhezri) [inline Capstone/API/APIManager.m]:
```suggestion
getArrayOfCollegeNews

File: Capstone/News/NewsViewController.m
Comment (skhezri) [inline Capstone/News/NewsViewController.m]: lets add a descriptive var for the number 4 so a reviewer understands what it represents!

**Comment by skhezri** [comment]
This is a great PR Abel! Thank you for keeping it small and adding a clear and concise summary and test plan. I've added some nit comments but otherwise looks good to go! Accepting to unblock but try and fix the typo error before merging 😊

**Review by skhezri — APPROVED** [review]

File: Capstone/Models/CollegeNews.m
Comment (angelacaq) [inline Capstone/Models/CollegeNews.m]: think you should be able to just do
```
dictionary[@"title"] != null
```
right?

File: Capstone/News/NewsViewController.m
Comment (angelacaq) [inline Capstone/News/NewsViewController.m]: 🎉🎉

File: Capstone/News/NewsViewController.m
Comment (angelacaq) [inline Capstone/News/NewsViewController.m]: Let's clarify the naming so that what `relation` is is clearer later one in this method
```suggestion
    PFRelation *likesRelation = [currentUser relationForKey:@"likes"];
```

File: Capstone/News/NewsViewController.m
Comment (angelacaq) [inline Capstone/News/NewsViewController.m]: another option for appending strings:
```
[NSString stringWithFormat:@"%@,", collegeSite];
```

This gets us the same string: `%@` will be filled in by `collegeSite`, and `,` is appended too without having to do another manual append

**Review by angelacaq — COMMENTED** [review]
This PR is also great — thanks for keeping them small!

Logic looks overall good, so like Sophia, I just have some nits to resolve. That's a good sign: it means the implementation is solid :)

File: Capstone/API/APIManager.h
Comment (AbelKel) [reply Capstone/API/APIManager.h]: Resolved

File: Capstone/API/APIManager.m
Comment (AbelKel) [reply Capstone/API/APIManager.m]: Since we're inputting domains of queried colleges using "(NSString *)collegeWebsite" isn't better to continue working with the previous code?

File: Capstone/News/NewsViewController.m
Comment (AbelKel) [reply Capstone/News/NewsViewController.m]: Resolved

File: Capstone/Models/CollegeNews.m
Comment (AbelKel) [reply Capstone/Models/CollegeNews.m]: {          
   "title":"Some news",
   "description":null,
   "urlToImage":"www.sth.com"
}

That was what I also originally assumed. However, in NSDictionary, you can only represent "null" values with NSNull.
https://developer.apple.com/documentation/foundation/nsdictionary?language=objc

File: Capstone/News/NewsViewController.m
Comment (AbelKel) [reply Capstone/News/NewsViewController.m]: Neat idea

Comment (AbelKel) [reply Capstone/News/NewsViewController.m]: Implemented!

**Pull request merged**
